modules/aggs-matrix-stats/src/test/java/org/elasticsearch/search/aggregations/matrix/stats/MatrixStatsAggregatorTests.java

```diff
@@ -53,10 +53,12 @@ public void testNoData() throws Exception {
                     .fields(Collections.singletonList("field"));
                 InternalMatrixStats stats = search(searcher, new MatchAllDocsQuery(), aggBuilder, ft);
                 assertNull(stats.getStats());
+                assertFalse(MatrixAggregationInspectionHelper.hasValue(stats));
             }
         }
     }
 
+    @AwaitsFix(bugUrl = "Test fails due to mismatched variance after reduction, unclear why.")
```

Comment:
I think this should link to an open github issue

Reply:
:+1: